_posts/2023-06-26-trino-fest-2023-salesforce.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: "Trino Fest 2023 recap: Anomaly detection for Salesforce’s production data using Trino"
+author: "Tuli Nivas, Geeta Shankar, and Ryan Duan"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/Salesforce.png
+show_pagenav: false
+---
+
+Rolling into our next presentation of Trinofest 2023, we're excited to bring you
+Tuli Navas and Geeta Shankar's talk from the Performance Engineering Team of
+Salesforce. They provide numerous reasons for why they need Trino and
+further continue explaining how it is essential for anomaly detection within
+their data. It's an insightful talk of how expeditious Trino can be that you definitely
+don't want to miss.
+
+<!--more-->
+
+{% youtube nFuqpb2GjVI %}
+
+## Recap
+
+Salesforce provides customer relationship management software and applications
+focused on sales, customer service, marketing automation, e-commerce, analytics,
+and application development. They host hundreds of thousands of customers that
+generate millions of transactions per day. As Tuli said, "Trino has been a
+one-stop shop for analytics". Trino allows them to do a multitude
+of things such as, querying all the data quickly and efficiently, and making it
+accessible in an optimal, easy, and timely fashion.
+
+Trino has helped Salesforce get more value from their production data by
+accelerating their access to it making their decision making faster and improved
+many other things. For years, they used Splunk for all their production data,
+but after switching to Trino, they have had numerous improvements:
+
+* Reducing their team's analytics cost
+* Improving their cost-to-serve
+* A 194% improvement in time it takes to run the same query
+* Provide a SLA of 20 minute latency on all production logs
+* Can retain and access data up to 2 years compared to Splunk's 30 days
+* Need fewer queries to run creating a smaller footprint
+* Able to create tables and views for temporary data storage and analytics
```

Comment:
For this entire list, let's make sure each bullet point is parallel in terms of structure. Let's stick to what you've done to the first two, with a continuous tense verb at the front. So maybe "Able to create tables..." turns into "Creating tables..."